Spelling-Correct in index.html title

Zoomhawk opened the pull request:
## Describe your changes
There is a spelling mistake in "docs/index.html" file at the title of the website, So i have just changed it from "Kubscape" to "Kubescape"
![Screenshot from 2022-09-16 02-53-24](https://user-images.githubusercontent.com/91390132/190520893-c35cc36c-1fdd-4621-a89e-fe6b83d67af9.png)
![Screenshot from 2022-09-16 02-53-30](https://user-images.githubusercontent.com/91390132/190520899-25ea4356-c600-4428-a61d-89ab84d9fb5d.png)

## Screenshots - If Any (Optional)

## This PR fixes:

* Resolved #

## Checklist before requesting a review
<!-- put an [x] in the box to get it checked -->

- [x] My code follows the style guidelines of this project
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

Sahocan6363 commented:
Aboneliğimi iptalini yaparmisiniz rica ediyorum bir daha bu mesaj görmek
istemiyorum rahtsiz oluyorum

16 Eyl 2022 Cum 01:40 tarihinde Harsh ***@***.***> şunu yazdı:

> Describe your changes
>
> There is a spelling mistake in "docs/index.html" file at the title of the
> website, So i have just changed it from "Kubscape" to "Kubescape"
> [image: Screenshot from 2022-09-16 02-53-24]
> <https://user-images.githubusercontent.com/91390132/190520893-c35cc36c-1fdd-4621-a89e-fe6b83d67af9.png>
> [image: Screenshot from 2022-09-16 02-53-30]
> <https://user-images.githubusercontent.com/91390132/190520899-25ea4356-c600-4428-a61d-89ab84d9fb5d.png>
> Screenshots - If Any (Optional) This PR fixes:
>
>    - Resolved #
>
> Checklist before requesting a review
>
>    - My code follows the style guidelines of this project
>    - I have commented on my code, particularly in hard-to-understand areas
>    - I have performed a self-review of my code
>    - If it is a core feature, I have added thorough tests.
>    - New and existing unit tests pass locally with my changes
>
> *Please open the PR against the dev branch (Unless the PR contains only
> documentation changes)*
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/kubescape/kubescape/pull/822
> Commit Summary
>
>    - 1e4dc49
>    <https://github.com/kubescape/kubescape/pull/822/commits/1e4dc49bbbe37ddf72f6b4891407b058a03cc8c3>
>    Spelling-Correct in index.html title
>
> File Changes
>
> (1 file <https://github.com/kubescape/kubescape/pull/822/files>)
>
>    - *M* docs/index.html
>    <https://github.com/kubescape/kubescape/pull/822/files#diff-b04b38d4e36f7a7171aeb211bf933eaf36d41d9866ebbc3639f673f84dc350ae>
>    (2)
>
> Patch Links:
>
>    - https://github.com/kubescape/kubescape/pull/822.patch
>    - https://github.com/kubescape/kubescape/pull/822.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kubescape/kubescape/pull/822>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZP4X3DM7QAXK5ZNLFCTED3V6OQUXANCNFSM6AAAAAAQN2H6DU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

Zoomhawk commented:
Ahh! Unable to understand your language

On Sun, 25 Sep 2022, 5:19 am Sahocan6363, ***@***.***> wrote:

> Aboneliğimi iptalini yaparmisiniz rica ediyorum bir daha bu mesaj görmek
> istemiyorum rahtsiz oluyorum
>
> 16 Eyl 2022 Cum 01:40 tarihinde Harsh ***@***.***> şunu yazdı:
>
> > Describe your changes
> >
> > There is a spelling mistake in "docs/index.html" file at the title of the
> > website, So i have just changed it from "Kubscape" to "Kubescape"
> > [image: Screenshot from 2022-09-16 02-53-24]
> > <
> https://user-images.githubusercontent.com/91390132/190520893-c35cc36c-1fdd-4621-a89e-fe6b83d67af9.png
> >
> > [image: Screenshot from 2022-09-16 02-53-30]
> > <
> https://user-images.githubusercontent.com/91390132/190520899-25ea4356-c600-4428-a61d-89ab84d9fb5d.png
> >
> > Screenshots - If Any (Optional) This PR fixes:
> >
> > - Resolved #
> >
> > Checklist before requesting a review
> >
> > - My code follows the style guidelines of this project
> > - I have commented on my code, particularly in hard-to-understand areas
> > - I have performed a self-review of my code
> > - If it is a core feature, I have added thorough tests.
> > - New and existing unit tests pass locally with my changes
> >
> > *Please open the PR against the dev branch (Unless the PR contains only
> > documentation changes)*
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/kubescape/kubescape/pull/822
> > Commit Summary
> >
> > - 1e4dc49
> > <
> https://github.com/kubescape/kubescape/pull/822/commits/1e4dc49bbbe37ddf72f6b4891407b058a03cc8c3
> >
> > Spelling-Correct in index.html title
> >
> > File Changes
> >
> > (1 file <https://github.com/kubescape/kubescape/pull/822/files>)
> >
> > - *M* docs/index.html
> > <
> https://github.com/kubescape/kubescape/pull/822/files#diff-b04b38d4e36f7a7171aeb211bf933eaf36d41d9866ebbc3639f673f84dc350ae
> >
> > (2)
> >
> > Patch Links:
> >
> > - https://github.com/kubescape/kubescape/pull/822.patch
> > - https://github.com/kubescape/kubescape/pull/822.diff
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/kubescape/kubescape/pull/822>, or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AZP4X3DM7QAXK5ZNLFCTED3V6OQUXANCNFSM6AAAAAAQN2H6DU
> >
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kubescape/kubescape/pull/822#issuecomment-1257084891>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVZIBNGLW5KKPFU6AY2QGY3V76HPJANCNFSM6AAAAAAQN2H6DU>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>
